packages/swingset-theme-hashicorp/src/components/side-nav/toggle-button.tsx

```diff
@@ -0,0 +1,22 @@
+import { IconChevronRight24 } from '@hashicorp/flight-icons/svg-react/chevron-right-24'
+import { cx } from 'class-variance-authority'
+
+type ToggleButtonProps = {
+ toggle: () => void
+ isOpen: boolean
+}
+
+export function ToggleButton({ toggle, isOpen }: ToggleButtonProps) {
+ return (
+  <button className="ss-fixed ss-left-72 ss-p-2 ss-top-6" onClick={toggle}>
+   <div
+    className={cx(
+     'ss-bg-surface-faint ss-border-faint ss-border-2 ss-rounded-full ss-transition-transform ss-shadow-md ss-p-1 ss-bg-opacity-60 ss-backdrop-blur-sm',
+     !isOpen && 'ss--rotate-180'
+    )}
+   >
```

Comment:
Do we need this nested `div`? I think many of these classes can be applied to the `button`, and we can apply the rotation to only the icon to avoid the shadow inverting the backdrop shadow.

Reply:
+1 here, and leaving a visual for myself

Current:
<img width="98" alt="CleanShot 2023-06-23 at 13 03 06@2x" src="https://github.com/hashicorp/swingset/assets/26389321/ae7a586c-29d7-4fec-9560-5adf23adba5b">

Reply:
Good catch, didn't even realize that. Will iterate